src/js/builtins/JSBufferPrototype.ts

```diff
@@ -9,45 +9,79 @@ interface BufferExt extends Buffer {
 }
 
 export function setBigUint64(this: BufferExt, offset, value, le) {
+  if (offset === undefined) offset = 0;
```

Comment:
```suggestion
  offset ??= 0;
```

Reply:
`a ??= b` also assigns if `a` is `null` which i do not want to do in this case (its emulating setting a default parameter value)